packages/core/src/connectors/index.ts

```diff
@@ -21,81 +22,69 @@ const loadConnectors = async () => {
   }
 
   const {
-    values: { additionalConnectorPackages },
+    values: { connectorDirectory },
   } = envSet;
 
-  const connectorPackages = [...defaultConnectorPackages, ...additionalConnectorPackages];
-
-  // eslint-disable-next-line @silverhand/fp/no-mutation
-  cachedConnectors = await Promise.all(
-    connectorPackages.map(async (packageName) => {
-      // eslint-disable-next-line no-restricted-syntax
-      const { default: createConnector } = (await import(packageName)) as {
-        default: CreateConnector<AllConnector>;
-      };
-      const rawConnector = await createConnector({ getConfig: getConnectorConfig });
-      validateConnectorModule(rawConnector);
-
-      const connector: LoadConnector = {
-        ...defaultConnectorMethods,
-        ...rawConnector,
-        validateConfig: (config: unknown) => {
-          validateConfig(config, rawConnector.configGuard);
-        },
-      };
-      // eslint-disable-next-line unicorn/prefer-module
-      const packagePath = resolvePackagePath(packageName, __dirname);
-
-      // For relative path logo url, try to read local asset.
-      if (
-        packagePath &&
-        !connector.metadata.logo.startsWith('http') &&
-        existsSync(path.join(packagePath, '..', connector.metadata.logo))
-      ) {
-        const data = readFileSync(path.join(packagePath, '..', connector.metadata.logo));
-        // eslint-disable-next-line @silverhand/fp/no-mutation
-        connector.metadata.logo = `data:image/svg+xml;base64,${data.toString('base64')}`;
-      }
-
-      if (
-        packagePath &&
-        connector.metadata.logoDark &&
-        !connector.metadata.logoDark.startsWith('http') &&
-        existsSync(path.join(packagePath, '..', connector.metadata.logoDark))
-      ) {
-        const data = readFileSync(path.join(packagePath, '..', connector.metadata.logoDark));
-        // eslint-disable-next-line @silverhand/fp/no-mutation
-        connector.metadata.logoDark = `data:image/svg+xml;base64,${data.toString('base64')}`;
-      }
-
-      if (
-        packagePath &&
-        connector.metadata.readme &&
-        existsSync(path.join(packagePath, '..', connector.metadata.readme))
-      ) {
-        // eslint-disable-next-line @silverhand/fp/no-mutation
-        connector.metadata.readme = readFileSync(
-          path.join(packagePath, '..', connector.metadata.readme),
-          'utf8'
-        );
-      }
+  if (!existsSync(connectorDirectory)) {
+    return [];
+  }
 
-      if (
-        packagePath &&
-        connector.metadata.configTemplate &&
-        existsSync(path.join(packagePath, '..', connector.metadata.configTemplate))
-      ) {
-        // eslint-disable-next-line @silverhand/fp/no-mutation
-        connector.metadata.configTemplate = readFileSync(
-          path.join(packagePath, '..', connector.metadata.configTemplate),
-          'utf8'
-        );
+  const connectorFolders = await readdir(connectorDirectory);
```

Comment:
Would there also be files instead of folders?

Reply:
This is the folder for all the connectors.